docs/docs/developers/contracts/writing_contracts/storage/notes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Notes
+---
+
+Some optionally background resources on notes can be seen here:
+- [High level network architecture](../../../../learn/about_aztec/technical_overview#high-level-network-architecture), specifically the Private Execution Environment
+- [Transaction lifecycle (simple diagram)](../../../../learn/concepts/transactions#simple-example-of-the-private-transaction-lifecycle)
+- [Public and Private state](../../../../learn/concepts/hybrid_state/main)
+
+Outside of this there are many more details to discover that will be skipped over for now:
+- Private and public contexts
+- Encryption keys and events
+- Oracle's role in using notes
+- Value Serialization/Deserialization
+
+This explainer will go through some examples that touch on what has been covered [here](../../../../learn/concepts/storage/notes).
+
+#### Some code context
+The way Aztec (`aztec-packages` monorepo) benefits from the Noir language is via three important components:
+- `Aztec-nr` - a Noir framework enabling contracts on Aztec, written in Noir. Includes useful Note implementations.
+- `noir contracts` - , example Aztec contracts
+- `noir-protocol-circuits` - a crate containing essential circuits for the protocol (public circuits and private wrappers)
+
+A lot of what we will look at will be in [aztec-nr/aztec/src/note](https://github.com/AztecProtocol/aztec-packages/tree/#include_aztec_version/noir-projects/aztec-nr/aztec/src/note), specifically the lifecycle and note interface.
+Looking at the noir circuits in these components, you will see references to the distinction between public/private execution and state.
+
+### Lifecycle functions
+Inside the [lifecycle](
+https://github.com/AztecProtocol/aztec-packages/tree/#include_aztec_version/noir-projects/aztec-nr/aztec/src/note/lifecycle.nr) circuits we see the functions to create and destroy a note, implemented as insertions of note hashes and nullifiers respectively.
+
+We also see a function to create a note hash from the public context, a way of creating a private variable from a public call (run in the sequencer).
```

Comment:
What are these functions? How do they help devs? How can a dev use them?

Reply:
Same as previous comment. Will expand in next iteration.

Reply:
After some initial replies here, I sought clarity and [this](https://github.com/AztecProtocol/aztec-packages/blob/master/docs/HOW_WE_WRITE_DOCS.md) helped.
Following from [this](https://github.com/AztecProtocol/aztec-packages/pull/4690/files/513d9c305fae28ee4bc4f43a9ba27fe214456158#r1505429093) I don't think everyone is on the same page on where diataxis meets learn/build, so lets resolve out of comments and I can update back here once clear.

Reply:
Added examples to the functions mentioned.